coalib/output/printers/LogPrinter.py

```diff
@@ -1,21 +1,37 @@
+import sys
 import traceback
+import logging
 
 from pyprint.ColorPrinter import ColorPrinter
 
-from coalib.output.printers.LOG_LEVEL import LOG_LEVEL, LOG_LEVEL_COLORS
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
 from coalib.processes.communication.LogMessage import LogMessage
 
 
+class PrinterHandler(logging.StreamHandler):
+    """
+    Handler for logging which writes messages into stdout.
+    """
+
+    def emit(self, record):
+        msg = self.format(record)
+        sys.stdout.write(msg + "\n")
+
+
 class LogPrinter:
     """
+    This class is deprecated and will be soon removed. To get logger use
+    loggin.getLogger('coala.colored') for colored logger and
```

Comment:
The logger has much less context. Also, as it is right now, it always writes to sys.stdout, so it cannot get pipped. I'll configure list log printer, and it will not be used with colours, and can be used if you want something to be piped.

One idea here, is to configure loggers to be called as `getLogger(__name__)`. Then configure the respective "module level" loggers, this way it can decide which kind of logger needs to be used based on the module it is in.